src/VirtoCommerce.BackInStockModule.Data/Models/BackInStockSubscriptionEntity.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using VirtoCommerce.BackInStockModule.Core.Models;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Domain;
+
+namespace VirtoCommerce.BackInStockModule.Data.Models
+{
+    public sealed class BackInStockSubscriptionEntity : AuditableEntity,
+        IDataEntity<BackInStockSubscriptionEntity, BackInStockSubscription>
```

Comment:
Don't break this line

Reply:
that line is normally broken. its too long.

Reply:
It's 137 characters.
By the way, remove `sealed`